CHANGELOG.md

```diff
@@ -1,11 +1,49 @@
-## Next release
-
-This release includes changes to amtool which are not fully backwards compatible with the previous
-amtool version (#1798) related to backup and import of silences.  If a backup of silences is created using 
-a previous version of amtool (v0.16.1 or earlier), it is possible that not all silences can be correctly imported using a later version of amtool.
-
+## 0.17.0 / 2019-05-02
+
+This release includes changes to amtool which are not fully backwards
+compatible with the previous amtool version (#1798) related to backup and
+import of silences. If a backup of silences is created using a previous
+version of amtool (v0.16.1 or earlier), it is possible that not all silences
+can be correctly imported using a later version of amtool.
+
+Additionally, the groups endpoint that was dropped from api v1 has been added
+to api v2. The default for viewing alerts in the UI now consumes from this
+endpoint and displays alerts grouped according to the groups defined in the
+running configuration. Custom grouping is still supported.
+
+This release has added two new flags that may need to be tweaked. For people
+running with a lot of concurrent requests, consider increasing the value of
+`--web.get-concurrency`. An increase in 503 errors indicates that the request
+rate is exceeding the number of currently available workers. The other new
+flag, `--web.timeout`, may also need modification from its default value.
+
+* [CHANGE] Modify the self-inhibition prevention semantics (#1873)
 * [CHANGE] Make api/v2/status.cluster.{name,peers} properties optional for Alertmanager with disabled clustering (#1728)
+* [FEATURE] Add groups endpoint to v2 api (#1791)
+* [ENHANCEMENT] Set HTTP headers to prevent asset caching (#1817)
+* [ENHANCEMENT] API returns current silenced/inhibited state of alerts (#1733)
+* [ENHANCEMENT] Introduce concurrency limit for GET requests and a general timeout for HTTP (#1743)
```

Comment:
Reading the modified intro text above, perhaps this should just be

`[ENHANCEMENT] Introduce configurable concurrency limit for GET requests (#1743)`

And then add a separate
`[FEATURE] Optional timeout for HTTP requests  (#1743)`

After all, the concurrency limit is only for GET, and the timeout is for all HTTP requests. We should probably not conflate them.

Reply:
"Enhancement" sounds like something you don't have to do anything about (usually), things just get better (which one might argue is true for the concurrency limit). However, the timeout is something you have to switch on and what you might or might not want. Thus, I'd count it as a "feature".